code/datums/outfits/costumes/halloween.dm

```diff
@@ -0,0 +1,115 @@
+/decl/hierarchy/outfit/h_masked_killer
+	name = "Costume - Masked Killer"
+	uniform = /obj/item/clothing/under/overalls
+	shoes = /obj/item/clothing/shoes/white
+	gloves = /obj/item/clothing/gloves/sterile/latex
+	mask = /obj/item/clothing/mask/surgical
+	head = /obj/item/clothing/head/welding
+	suit = /obj/item/clothing/suit/storage/apron
+	r_hand = /obj/item/weapon/material/twohanded/fireaxe/foam
+
+/decl/hierarchy/outfit/masked_killer/post_equip(var/mob/living/carbon/human/H)
+	var/victim = get_mannequin(H.ckey)
+	for(var/obj/item/carried_item in H.get_equipped_items(TRUE))
+		carried_item.add_blood(victim) //Oh yes, there will be blood.. just not blood from the killer because that's odd. //If I knew how to make fake blood, I would
```

Comment:
Tomato splatter can fake blood, yes?

Reply:
I don't think that's up, but I can look around.